package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "protractor-sync",
-  "version": "1.0.6",
+  "version": "2.0.0",
```

Comment:
Syncing this to the protractor version makes a lot of sense. I was looking at https://github.com/teerapap/grunt-protractor-runner/blob/eb9e67fd0cd1f9bf4d93e66654967b110d793693/package.json just now and am also thinking we should be more flexible in how we reference protractor and other deps to prevent multiple different versions from getting included in projects that try to use this library.